src/Kriti/CustomFunctions.hs

```diff
@@ -0,0 +1,86 @@
+module Kriti.CustomFunctions (basicFunctions) where
```

Comment:
I would probably also export the functions themselves, not just the set, since then users can mix and match how they like.

Reply:
Sure